frontend/public/components/routes/create-route.tsx

```diff
@@ -216,6 +281,20 @@ export class CreateRoute extends React.Component<null, CreateRouteState> {
       Redirect: 'Redirect',
     };
 
+    const alternateServicesList = _.map(alternateServices, (entryData, index) => {
+      return <div className="co-create-secret__form-entry-wrapper" key={entryData.key}>
+        {!_.isEmpty(alternateServices) && <div className="co-create-secret-form__link--remove-entry">
```

Comment:
We shouldn't reference `co-create-secret-form` classes directly. Those should only appear in the create secret form. We should generalize them if we want to reuse them.